home.html

```diff
@@ -0,0 +1,155 @@
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </div>
+        <div class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/music-media.jpg" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Advertising & Media</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </div>
+        <div class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/program_icon_03.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Building Connection</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </div>
+        <div class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/program_icon_02.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Drum Brands Presentaion</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </div>
+        <div class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/program_icon_01.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Drum Brands</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </div>
+        </div>
+        <a class="see-more" href="#">SEE THE WHOLE PROGRAMS</a>
+    </section>
+
+    <section class="speaker-section">
+        <h2 class="speaker-heading">Featured Speakers</h2>
+        <div class="program-bar"></div>
+        <div class="speaker-container" id="speakers" >
+        </div>
+    </section>
+    <hr>
+
+    <section class="sponsors" id="sponsor">
+        <h2 class="sponsors-heading">Sponsors</h2>
+        <div class="program-bar"></div>
+        <div class="sponsor-logos">
+            <img src="img/airbnb.png" alt="airbnb logo">
+            <img src="img/google-logo.jpg" alt="agoogle logo">
+            <img src="img/Yamaha-Logo.png" alt="yamaha logo">
+            <img src="img/spotify-logo.jpg" alt="spotify logo">
+            <img src="img/maverick-city.jpg" alt="maverick-city logo">
+            <img src="img/africa-logo.jpg" alt="africa logo">
+            <img src="img/apple-pay-logo.jpg" alt="apple pay logo">
+        </div>
```

Comment:
- [ ] Kindly align the logos so that they look like the given design, using `flexbox` will help you achieve this easily. Also reducing the image size for all the logos to a smaller one will make them fit the screen prperly. Lastly you could use the provided color swatches for the background instead of the white background currently being used.

Your design
![image](https://user-images.githubusercontent.com/81932478/196773011-067d10e4-c33a-44b9-8e30-93ba13ba2f68.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/196773097-3be77e68-0cf5-42e3-b9fa-b5172863a74c.png)